### ISSUE TITLE
Quarkus Admin Manager

### ISSUE THREAD

**Issue by AnakinPt** [quarkusio/quarkus#22522]
I'm a devops who develops and manages multiple quarkus services, in multiple clusters in AWS, using ECS Fargate.

I would like to have a way, to manage my microservices, so I can do some operational changes, without the need to create new ECS Tasks or killing running services. One example of this will the the update of the log level of the application or a class in a microservice. I need a system, that knows where all my Quarkus Applications are running, and can connect to them, to send requests to change the log level of them.

Scenario: 1 ECS Service can be running 10 tasks, each being a quarkus application. I can only access the service using a NLB, or even not having access to the service at all, from outside the network.

Bottom Line: I need a similar feature of Spring Boot Admin https://github.com/codecentric/spring-boot-admin for Quarkus. Where an application registers itself in the Admin server and then, we can perform actions on it.

**Comment by melloware** [comment]
I would love this too but I feel just like SBA which is an open source project not sponsored by Spring someone would have to write the same thing for Quarkus.

**Comment by geoand** [comment]
I am going to close this as a duplicate in favor of https://github.com/quarkusio/quarkus/issues/18759
